geckodriver.log

```diff
@@ -669,3 +669,55 @@ JavaScript error: resource://gre/actors/AutoCompleteChild.jsm, line 125: Error:
 
 ###!!! [Child][RunMessage] Error: Channel closing: too late to send/recv, messages will be lost
```

Comment:
I missed this last time, but this is another file that there's no point tracking , so we could add this to the `.gitignore` and run `git rm --cached geckodriver.log` to remove the existing file